.github/workflows/releaser.yml

```diff
@@ -0,0 +1,36 @@
+# .github/workflows/release.yml
+name: goreleaser
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: setup dependencies
+        uses: actions/setup-go@v2
+
+      - name: Login to GHCR
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Publich ref name
+        run: echo "Build for tag ${{ github.ref_name }}"
```

Comment:
does this work? in the past I've had to use this way:

https://github.com/flashbots/mev-boost/blob/develop/.github/workflows/release.yaml#L20-L26

Reply:
Yes, check this execution log https://github.com/ferranbt/suave-geth/actions/runs/7446372136/job/20256482872